presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -148,6 +148,9 @@ class SystemConfig : public ConfigBase {
 
   static SystemConfig* instance();
 
+  template <typename T>
+  std::string veloxProperty(const std::string& kKey, const T& kDefault) const;
```

Comment:
naming: kKey -> key, kDefault -> defaultValue

naming: perhaps, veloxProperty -> get

Please, add a comment to document this method.